MotionDetection/MotionDetection.cpp

```diff
@@ -66,6 +66,8 @@ namespace WPEFramework {
             Register("setSensitivity", &MotionDetection::setSensitivity, this);
             Register("getSensitivity", &MotionDetection::getSensitivity, this);
             Register("getLastMotionEventElapsedTime", &MotionDetection::getLastMotionEventElapsedTime, this);
+            Register("setMotionEventsActivePeriod", &MotionDetection::setMotionEventsActivePeriod, this);
+            Register("getMotionEventsActivePeriod", &MotionDetection::getMotionEventsActivePeriod, this);
```

Comment:
Pls register them for version 2. You can refer to this commit on how to add versioning. https://github.com/rdkcentral/rdkservices/commit/3692632373e8e82dba92bec56f9e6082b430a829#diff-f6cd28bb8911a0253a4601f823d3777089aecd54eb214bc6cdc227961da7b13f

Reply:
Hi Anand. These are actually V1 APIs.